bundles/org.openhab.ui/web/src/components/widgets/widget-mixin.js

```diff
@@ -20,13 +20,17 @@ expr.jsep.plugins.register(jsepRegex, jsepArrow, jsepObject, jsepTemplate)
 
 expr.addUnaryOp('@', (itemName) => {
   if (itemName === undefined) return undefined
-  const itemState = store.getters.trackedItems[itemName]
-  return itemState.displayState || itemState.state
+  const item = store.getters.trackedItems[itemName]
+  return item.displayState || item.state
```

Comment:
I'm too late, but be careful with this syntax with the OR operator - here if `item.displayState === 0`  you do fallback to `item.state` which is not what you want.
Nice feature btw!